docs/3.0.0-beta.x/guides/external-data.md

```diff
@@ -0,0 +1,96 @@
+# Fetching external data
+
+This guide will explain how to fetch data in an external service and use it in your app.
```

Comment:
This guide explains how to fetch data from an external service to use it in your app.